woocommerce/woocommerce
# [3.6] Add manage suggestions link into browse-all metabox footer

### All Submissions:


### Changes proposed in this Pull Request:
Adds a "Manage suggestions" link to the product list metabox, linking to settings.

### How to test the changes in this Pull Request:
1. Edit a product. Scroll down to metabox, click `Get more options` tab.
2. Notice new `Manage suggestions` link at bottom right.

Note that the link destination (the new settings section) is implemented in separate PR #23218 .

<img width="923" alt="Screen Shot 2019-04-08 at 9 22 37 AM" src="https://user-images.githubusercontent.com/4167300/55690554-313e6180-59e7-11e9-9dcc-de858f071035.png">
